src/sentry/integrations/jira/client.py

```diff
@@ -59,15 +55,46 @@ def request(self, method, path, data=None, params=None, **kwargs):
             jwt=encoded_jwt,
             **(url_params or {})
         )
-        return self._request(method, path, data=data, params=params, **kwargs)
+        request_spec = kwargs.copy()
+        request_spec.update(dict(method=method, path=path, params=params))
+        return request_spec
+
+
+class JiraApiClient(ApiClient):
+    COMMENT_URL = '/rest/api/2/issue/%s/comment'
+    STATUS_URL = '/rest/api/2/status'
+    CREATE_URL = '/rest/api/2/issue'
+    ISSUE_URL = '/rest/api/2/issue/%s'
+    META_URL = '/rest/api/2/issue/createmeta'
+    PRIORITIES_URL = '/rest/api/2/priority'
+    PROJECT_URL = '/rest/api/2/project'
+    SEARCH_URL = '/rest/api/2/search/'
+    VERSIONS_URL = '/rest/api/2/project/%s/versions'
+    USERS_URL = '/rest/api/2/user/assignable/search'
+    SERVER_INFO_URL = '/rest/api/2/serverInfo'
+    ASSIGN_URL = '/rest/api/2/issue/%s/assignee'
+    TRANSITION_URL = '/rest/api/2/issue/%s/transitions'
+
+    def __init__(self, base_url, jira_style, verify_ssl):
```

Comment:
I think having some comment here explaining what a `jira_style` is will be helpful.

Reply:
Done.